pom.xml

```diff
@@ -1922,6 +1922,7 @@
               <JAVA_HOME>${test.java.home}</JAVA_HOME>
             </environmentVariables>
             <systemProperties>
+              <log4j.configuration>file:src/test/resources/log4j.properties</log4j.configuration>
```

Comment:
This seems to be necessary for some unexplained reason; without this, the removal of the explicit surefire configuration in the `java8-tests` POM leads to tons of log spam.